package.json

```diff
@@ -12,16 +12,31 @@
     "appsignal-diagnose": "./bin/diagnose"
   },
   "dependencies": {
-    "@opentelemetry/api": "^1.1.0",
-    "@opentelemetry/auto-instrumentations-node": "^0.32.0",
-    "@opentelemetry/sdk-node": "^0.30.0",
-    "@opentelemetry/sdk-trace-base": "^1.4.0",
-    "@prisma/instrumentation": "^4.3.1",
+    "@opentelemetry/api": "^1.2.0",
+    "@opentelemetry/sdk-node": "^0.33.0",
+    "@opentelemetry/sdk-trace-base": "^1.7.0",
+    "@opentelemetry/instrumentation-graphql": "^0.31.0",
+    "@opentelemetry/instrumentation-http": "^0.32.0",
+    "@opentelemetry/instrumentation-ioredis": "^0.32.1",
+    "@opentelemetry/instrumentation-koa": "^0.32.0",
+    "@opentelemetry/instrumentation-mysql2": "^0.32.0",
+    "@opentelemetry/instrumentation-mysql": "^0.31.1",
+    "@opentelemetry/instrumentation-pg": "^0.31.1",
+    "@opentelemetry/instrumentation-redis-4": "^0.33.0",
+    "@opentelemetry/instrumentation-redis": "^0.33.0",
+    "@opentelemetry/instrumentation-express": "^0.31.2",
+    "@prisma/instrumentation": "^4.4.0",
     "node-addon-api": "^3.1.0",
     "node-gyp": "^9.0.0",
     "tslib": "^2.0.3",
     "winston": "^3.6.0"
   },
+  "overrides": {
+    "@opentelemetry/api-metrics": "^0.33.0",
+    "@opentelemetry/core": "^1.7.0",
+    "@opentelemetry/instrumentation": "^0.33.0",
+    "@opentelemetry/semantic-conventions": "^1.7.0"
+  },
```

Comment:
> Requires the dependency bump in https://github.com/appsignal/appsignal-nodejs/pull/742, so it's built on top of it -- ignore the other commits.

You know you can change the base branch to your other PR so these commits don't show in this PR?

Reply:
Ah, should've done that, yes 😅